contracts/test/strategies/balancerMetaStablePool.fork-test.js

```diff
@@ -174,6 +174,15 @@ describe("ForkTest: Balancer MetaStablePool rETH/WETH Strategy", function () {
         auraRewardPool
       );
     });
+
+    // Un-skip once we re-deploy the strategy
+    it.skip("Shouldn't be able to cache assets twice", async function () {
```

Comment:
How are we going to make sure we do this?

Reply:
How about like this :)
https://github.com/OriginProtocol/origin-dollar/pull/1889/commits/74fd116b39c4df5174c427d17eb524d9bfd4c770